modules/serverless-rn-1-37-0.adoc

```diff
@@ -92,3 +126,10 @@ kn: /lib64/libc.so.6: version `GLIBC_2.33' not found (required by kn)
 ----
 
 * As of {ServerlessProductName} 1.37 release, the `kn` client binary downloaded from the *Command Line Tools* page in the {ocp-product-title} web console is not signed with the Red{nbsp}Hat certificate for macOS and Windows platforms. This issue affects the binaries available directly through the {ocp-product-title} console. To obtain properly signed binaries, download them from the link:https://mirror.openshift.com/pub/cgw/serverless/[Official OpenShift Serverless downloads mirror] instead.
+
+[id="known-issues-osl-1-37-0_{context}"]
+=== {ServerlessLogicProductName}
+
+* In disconnected cluster environments, the `logic-swf-builder-rhel8` image always attempts to download the `plexus-utils-1.1.jar` dependency during the build process. Because external network access is restricted in disconnected setups, this behavior can cause build failures or timeouts. 
+
+* If you apply a `SonataFlow` custom resource (CR) to an OpenShift cluster and the first `SonataFlowBuild` fails for any reason, the Operator does not create the workflow deployment even after the build issue is resolved. As a result, the workflow remains undeployed until you manually re-apply or rebuild it.
```

Comment:
Suggestion to use the language as per [style guide](https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-known-issues):

```
Heading that summarizes the known issue
<Cause>. As a consequence, <consequence>.

To work around this problem, <workaround in imperative>. As a result, <result>.

TICKET-REFERENCE
```